src/services/__tests__/pagoPAProxyService.test.ts

```diff
@@ -177,6 +182,24 @@ describe("PagoPAProxyService#getPaymentInfo", () => {
     expect(res.kind).toEqual("IResponseErrorValidation");
   });
 
+  it("getPaymentInfoDefaultDecoder must validate correctly a PaymentProblemJson response", async () => {
```

Comment:
the test reads

`it getPaymentInfoDefaultDecoder must validate correctly a PaymentProblemJson response`

Reply:
fixed in 35258de. I don't now if now is better.